openedx/core/djangoapps/coursegraph/management/commands/dump_to_neo4j.py

```diff
@@ -90,11 +83,14 @@ def serialize_item(item):
 
         label = item.scope_ids.block_type
```

Comment:
Oh I see, you mean `label` is the `block_type`, Why can't we rename it to `item_block_type` just to be clear and readable.